ganache.js

```diff
@@ -1,14 +1,16 @@
 const ganache = require("ganache");
-require("dotenv").config()
+require("dotenv").config();
+
+const { GANACHE_PORT, GANACHE_SEED, GANACHE_CHAIN_ID } = process.env;
 
 const options = {
-  seed: process.env.SEED || "random_seed",
+  seed: GANACHE_SEED || "random_seed",
   db_path: "blockchain",
   account_keys_path: ".secrets",
-  chainId: process.env.CHAIN_ID || 1337
+  chainId: GANACHE_CHAIN_ID || 1337,
 };
 const server = ganache.server(options);
-const PORT = Number(process.env.PORT || 8545);
+const PORT = Number(GANACHE_PORT || 8545);
```

Comment:
Shouldn't the default port be `7545` and not `8545`? (I know you didn't change this part)

Reply:
Aah, I think the real default is `8545` but the localhost default is `7545`. So we can change it to `7545`!